README.md

```diff
@@ -137,27 +132,20 @@ Install-packageProvider -Name NuGet -Force
 Install-Module -Name AzureRM -Force
 ```
 
- 1. Run Microsoft Powershell.
- 1. Create the local parameters file by calling
-   ```
-   Invoke-Webrequest -Uri "https://raw.githubusercontent.com/teradici/deploy/master/azuredeploy.parameters.json" -OutFile "my.azuredeploy.parameters.json"
-   ```
- 3. Modify <samp>my.azuredeploy.parameters.json</samp> to include the necessary deployment parameters.
- 1. Run the following script, substituting username, password, resource group name, and desired region:
-
-```
-$spUsername = "<username>@<example>.com"
-$spPass = ConvertTo-SecureString "<password>" -AsPlainText -Force
-$cred = New-Object -TypeName pscredential -ArgumentList $spUsername, $spPass
-Login-AzureRMAccount -Credential $cred
-
-$azureRGName = "<rgname>"
-New-AzureRMResourceGroup -Name $azureRGName -Location "East US"
-New-AzureRMResourceGroupDeployment -DeploymentName "ad1" -ResourceGroupName $azureRGName -TemplateFile "https://raw.githubusercontent.com/teradici/deploy/master/azuredeploy.json" -TemplateParameterFile "my.azuredeploy.parameters.json"
-
-```
-
-If you do not want credentials in the file just go directly to <samp>Login-AzureAccount</samp> without the <samp>-Credential</samp> parameter and it will give you a prompt.
+1. Run Microsoft Powershell.
+1. Log into Azure using one of the following methods
```

Comment:
The SA's liked the 'pass a bunch of parameters through the command line' method like the second set of instructions I gave, where it's easy to fill in the blanks and hit 'go' essentially. I'm wondering if we should document that as well. Thoughts?

Reply:
Personally I think we should try to keep the README as short as needed. I can change the primary method for deploying it to use the command line parameters or add it in and keep both if you'd prefer.

Reply:
I think the default option should definitely be what you're documenting. I think the other method should be mentioned somewhere(tm). I don't know if it's here or some 'deployment for experts' section somewhere or what.